src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product" data-qa="card">
+      <div class="product__image">
+        <img
+          src="./images/imac.jpeg"
+          alt="Imac image"
+          class="image"
+        >
+      </div>
+      <div class="description product__description">
+        <div class="description__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+        <div class="description__id">
+          Код товара: 195434
+        </div>
+      </div>
+      <div class="rating product__rating">
+        <div class="stars">
```

Comment:
add stars--4 modifier, and write styles for all the possible modifiers in scss, so that changing modifier to stars--5 in html would change number of active stars accordingly